src/QMCWaveFunctions/LCAO/SoaAtomicBasisSet.h

```diff
@@ -56,6 +61,8 @@ struct SoaAtomicBasisSet
   std::vector<QuantumNumberType> RnlID;
   ///temporary storage
   VectorSoaContainer<RealType, 4> tempS;
+  struct SoaAtomicBSetMultiWalkerMem;
+  ResourceHandle<SoaAtomicBSetMultiWalkerMem> mw_mem_handle_;
```

Comment:
Please put the forward declaration at the beginning of this class.
and also briefly document `mw_mem_handle_`

Reply:
fixed